src/dash-adapter.js

```diff
@@ -202,19 +224,27 @@ export default class DashAdapter extends BaseMediaSourceAdapter {
     }
   }
 
+
+  _createAdapterFunctionBindings(): void {
```

Comment:
We can do this logic statically when we declare the `_adapterEventsBindings ` variable, no need to create method for it

Reply:
@dan-ziv what do you mean statically? we want to have the this bindings

Reply:
_adapterEventsBindings: { [name: string]: Function } = {
    [ShakaEvent.ERROR] : (event) => this._onError(event),
    ...
  };